package/libs/boringssl/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
```

Comment:
Use an SPDX license identifier instead.

Reply:
So, resolving without doing any changes. Really?

Reply:
Sorry, my bad. I might have misunderstood your comment. I had only changed OpenWrt.org to my name.

I've now added the SPDX identifier in the latest commit.